Code/training_team8_uzeros.py

```diff
@@ -0,0 +1,88 @@
+# Starting from sample code:
```

Comment:
I am not crazy about having 2 files that do almost the same thing.  The only difference is whether it imports `label_preprocessing_uzeros` or `label_preprocessing_uones` as base_label_transform.    I don't think this really justifies having 2 separate long files.  I am willing to accept this for our efficiency, but I wouldn't be surprised if Amir thinks it's ugly.

Reply:
Amir will not care. 
That being said my attempt to reduce copy-pasting was the common_training file. I'd intended to start copying the training files per model we need to train. If you want to go with another approach we'll need to set it up, but this should allow us to pursue separate models over the next few days without running into crazy merge conflicts.